libsnark/relations/constraint_satisfaction_problems/r1cs/r1cs.tcc

```diff
@@ -175,6 +175,7 @@ void r1cs_constraint_system<FieldT>::add_constraint(const r1cs_constraint<FieldT
 #ifdef DEBUG
     constraint_annotations[constraints.size()] = annotation;
 #endif
+    (void)annotation;
```

Comment:
Let's put this one in an `#else`
Something like:
```
#ifdef DEBUG
    constraint_annotations[constraints.size()] = annotation;
#else
    libff::UNUSED(annotation);
#endif
```

Reply:
Thanks.  I hadn't noticed the `libff::UNUSED`.  Switched to use that everywhere.